src/main/java/com/github/hiroyuki_sato/digdag/plugin/SshOperatorFactory.java

```diff
@@ -98,9 +96,13 @@ public TaskResult runTask()
                             .withRetryLimit(max_retry_limit);
 
                     try {
-                        retryExecutor.run(() -> {
+                        ssh = retryExecutor.run(() -> {
+                            SSHClient sshTmp = null;
```

Comment:
IntelliJ claimed `Variable sshTmp initializer 'null' is redundant.`

What do you think this change?


```diff
--- a/src/main/java/com/github/hiroyuki_sato/digdag/plugin/SshOperatorFactory.java
+++ b/src/main/java/com/github/hiroyuki_sato/digdag/plugin/SshOperatorFactory.java
@@ -97,9 +97,8 @@ public class SshOperatorFactory

                     try {
                         ssh = retryExecutor.run(() -> {
-                            SSHClient sshTmp = null;
                             try {
-                                sshTmp = new SSHClient();
+                                SSHClient sshTmp = new SSHClient();
                                 setupHostKeyVerifier(sshTmp);
                                 sshTmp.connect(host, port);
                                 return sshTmp;
```